src/toil/batchSystems/lsf.py

```diff
@@ -93,21 +93,23 @@ def getJobExitCode(self, lsfJobID):
                          "{}".format(job))
             process = subprocess.Popen(args, stdout=subprocess.PIPE,
                                        stderr=subprocess.STDOUT)
+            output = process.stdout.read().decode('ascii').replace("\n                     ", "")
```

Comment:
I am comfortable with changing encoding to utf-8, @drjrm3 what do you think? Any reason to stick with ascii?

Reply:
The line format is because of the actual output of `bjobs -l` formatting. It looks like:
```
..., Started on <node123>, 1 Task Started on node123, Status <RUN>, Memory...,
                                           (other reported metrics), (other reported metrics).....,
                                           (continue the report here), (continue the report here),
                                           Task <task-name> Done successfully..."
```
@drjrm3 wrote a pipeline in house that uses something similar for job monitoring and saw some issues with the number of spaces in line continuations. Perhaps he can comment on this a bit more?

Reply:
`utf-8` seems fine.

For the bunch of spaces + newline, it allows us to unfold each entry in `bjobs -l`. As it is without that transformation, we may not be able to detect things like `<DONE>` since it could be folded. By unwrapping that and removing the newline + large indent, it allows us to be sure we can capture the reported states to make decisions.